### ISSUE TITLE
Cannot find module 'internal/fs'

### ISSUE THREAD

**Issue by chai2010** [isaacs/node-graceful-fs#113]
```
node -v
v7.7.3

gitbook --version
CLI version: 2.3.0
GitBook version: 2.6.7

gitbook build
Error loading version latest: Error: Cannot find module 'internal/fs'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at evalmachine.<anonymous>:18:20
    at Object.<anonymous> (C:\Users\chai\.gitbook\versions\2.6.7\node_modules\npm\node_modules\graceful-fs\fs.js:11:1)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)

TypeError: Cannot read property 'commands' of null
```

**Comment by isaacs** [comment]
What version of graceful-fs?  I believe this was fixed in 4.0.0 or 4.1.0.

**Comment by zhuyali** [comment]
@chai2010 I have the same problem, how you fix it?
